Adyen/Components/Base/PaymentComponentData.swift

```diff
@@ -20,16 +20,24 @@ public protocol ShopperInformation {
 
 }
 
-/// Any object that holds shopper billing address information/
+/// Any object that holds shopper billing address information
 public protocol BillingAddressInformation {
```

Comment:
maybe add `billingAddress` and `socialSecurityNumber ` to `ShopperInformation` and make a default extension implementation with nil values.